plugins/modules/luks_device.py

```diff
@@ -217,6 +217,47 @@
             - "Will only be used on container creation."
         type: int
         version_added: '1.5.0'
+    perf_same_cpu_crypt:
+        description:
+            - "This option allows user to perform encryption using the same cpu that IO was submitted on."
```

Comment:
```suggestion
            - "Allows the user to perform encryption using the same CPU that IO was submitted on."
```

Reply:
```
╭─sirex at sirex-notebook in ~/ansible_collections/community/crypto/plugins/modules 22-04-02 - 15:13:21
╰○ grep 'This option allows' luks_device.py |wc
      8     116     815
╭─sirex at sirex-notebook in ~/ansible_collections/community/crypto/plugins/modules 22-04-02 - 15:13:28
╰○ grep 'Allows the user' luks_device.py |wc   
      0       0       0
```

So I'd better stick with default wording, but yep, I'll fix some english typos.

Reply:
Right now there are 4 instances, you are adding 4 more, so it's not really the "default wording". I personally don't think it's necessary to point out that an option is an option for every option, but that's also something that can be changed later (or kept).